src/Symfony/Component/HttpFoundation/RequestMatcher/IpsRequestMatcher.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpFoundation\RequestMatcher;
+
+use Symfony\Component\HttpFoundation\IpUtils;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestMatcherInterface;
+
+/**
+ * Checks the client IP of a Request.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+class IpsRequestMatcher implements RequestMatcherInterface
+{
+    private array $ips;
+
+    /**
+     * @param string[]|string $ips A specific IP address or a range specified using IP/netmask like 192.168.1.0/24
```

Comment:
This does not mention that those strings can contain a comma-delimited list of IPs or CIDR ranges too.